src/run_metrics_on_file.ts

```diff
@@ -15,56 +15,40 @@ import {
   PieceLevelMoveInfoMetric,
 } from './metrics/moves';
 import { PromotionMetric } from './metrics/promotions';
-import { RESULTS_PATH } from './queue';
 
 /**
  *
  * @param path
  * @returns
  */
 export async function main(path: string) {
+  // run analysis for the given path
   console.time('Total Execution Time');
-  await gameIterator(path, { 'Number of games analyzed': 0 });
+  const fileResults = await gameIterator(path);
   console.timeEnd('Total Execution Time');
 
   const now = new Date();
   const milliseconds = now.getMilliseconds();
 
   const analysisKey = `analysis_${now
     .toLocaleString()
-    .replace(/\/|,|:|\s/g, '_')}_${milliseconds}`;
-
-  let existingResults = {};
-  if (fs.existsSync(RESULTS_PATH)) {
-    const fileContent = fs.readFileSync(RESULTS_PATH, 'utf8');
-    if (fileContent !== '') {
-      existingResults = JSON.parse(fileContent);
-    }
-  }
-
-  console.log('sending results');
-
-  // TODO: Probably we need to read in the existing results in the queue server and merge them, as when there are multiple items in the queue
-  // this is going to be out of date
-  existingResults[analysisKey] = {
-    'Number of games analyzed': 0,
-  };
+    .replace(/\/|,|:|\s/g, '_')}_${milliseconds}_${randomUUID()}`;
```

Comment:
@bennyrubanov chgs

Reply:
no preference here. randomUUID good idea, but the date info was helpful so good that you kept